Gemfile

```diff
@@ -2,9 +2,9 @@
 
 source "https://rubygems.org"
 
-gem "activerecord", "~> 5.0.7"
-gem "mysql2", '~> 0.4.0'
+gem "activerecord", "~> 5.2.3"
```

Comment:
we are modifying the gemfile to be more current

Also was concerned nailing down a gem version when future versions of active record no longer require a particular version